server/lib/enrich/index.js

```diff
@@ -1,22 +1,18 @@
 'use strict';
 
-const path = require('path');
-
 const log = require('../logger');
 
-const MODULE_ID = path.relative(process.cwd(), module.id) || require(path.resolve('./package.json')).name;
-
 module.exports = exports = function enrich(content, format) {
 	if (!content.type && content.content_type) {
 		content.type = content.content_type;
 	}
 
-	if (Object.prototype.toString.call(content) === '[object Object]' && !(content instanceof Error) && content.type in exports) {
+	if (content instanceof Object && !(content instanceof Error) && content.type in exports) {
```

Comment:
Careful here because 
```js
Object.prototype.toString.call([]) === '[object Object]' // false
[] instanceof Object // true
```

not sure if arrays maybe want to be not allowed?